CONVENTIONS.md

```diff
@@ -116,12 +116,87 @@ Name screen coordinates `x` and `y`, rather than `left` and `top`.
+
+### Custom tags
+
+A few additional block tags are supported:
+
+ - `@apiNote`: API Notes. A few comments for users of the documented API.
+ - `@implSpec`: Implementation Specification. Tells how this method is implemented; usually avoided as yarn doesn't define implementations.
+ - `@implNote`: Implementation Notes. A few comments on the implementation.
+
+Feel free to use these tags and write under these sections.
+
+### Linking to other members
 
 Use `@link`, `@linkplain` and `@see` tags to refer to other parts of the code.
 
+You can use the simple name of a class when its import is assumed to exist.
+
+A class is assumed to be imported in the following scenarios:
+
+- If it is from the `java.lang` package
+- If it is from the same package as the currently documented class
+- If it is used as part of its API, such as in the signature of the class or its members (methods and fields). See Javadoc's definition of "use" in its [`-use` command line option specification](https://docs.oracle.com/en/java/javase/15/docs/specs/man/javadoc.html#options-for-javadoc).
+
+If it does not fulfill one of these scenarios, use the [full binary name](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/ClassLoader.html#binary-name), such as `com.google.common.collect.Lists` rather than simply `Lists`. Unlike class naming in enigma, do not use `/` to separate packages; use `.` instead.
+
+Use Yarn mappings when referencing Minecraft members, such as `net.minecraft.server.world.ThreadedAnvilChunkStorage` rather than `net.minecraft.class_3898`. The Javadoc task will warn if some links no longer work after a rename.
+
+<details>
+<summary>
+An example
+</summary>
+
+Assume this is the decompiled content in Enigma, which does not show imports:
+
+```java
+/**
+ * Assume this class is from the {@code net.example.stuff} package.
+ *
+ * <p>You can link to {@link Optional} as it's part of the class signature (type parameter bound).
+ *
+ * <p>You must fully qualify {@link net.example.stuff.basic.BasicStuffUser} when linking as it is not in
+ * any signature and is from a different package.
+ */
+public class Stuff<T extends Optional<?>> {
+  /**
+   * You can link to {@link Listener} with the simple name as it's part of a field's signature.
+   */
+  protected Listener listener;
+
+  /**
+   * You can link to {@link List} with the simple name as it's part of a method's signature.
+   *
+   * <p>You must fully qualify {@link net.example.util.UtilityClass} when linking because it is not part
+   * of any signature (even though it is used in code) and is from a different package.
+   */
+  public Stuff(List<Integer> opt) {
+    UtilityClass.callMethod(opt);
+  }
+}
```

Comment:
Use tabs for indentation:

```suggestion
public class Stuff<T extends Optional<?>> {
	/**
	 * You can link to {@link Listener} with the simple name as it's part of a field's signature.
	 */
	protected Listener listener;

	/**
	 * You can link to {@link List} with the simple name as it's part of a method's signature.
	 *
	 * <p>You must fully qualify {@link net.example.util.UtilityClass} when linking because it is not part
	 * of any signature (even though it is used in code) and is from a different package.
	 */
	public Stuff(List<Integer> opt) {
		UtilityClass.callMethod(opt);
	}
}
```

Reply:
Looks good